From 187c197a80b9d81f36b346fbca521a3eb06ee095 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Tue, 7 Feb 2023 16:47:01 +0000
Subject: [PATCH 1/2] d/control: Depend on polkitd | policykit-1, not just
 policykit-1

This allows colord to be installed without pulling in the transitional
package policykit-1.

Closes: #1025554, #1025379
---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 11a74489..726e67a1 100644
--- a/debian/control
+++ b/debian/control
@@ -77,7 +77,7 @@ Depends:
  acl,
  adduser,
  colord-data,
- policykit-1 (>= 0.103),
+ polkitd | policykit-1 (>= 0.103),
  ${misc:Depends},
  ${shlibs:Depends},
 Suggests:
-- 
2.39.1

